atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1449,22 +1488,41 @@ private void createTablesInternal(final Map<TableReference, byte[]> tableNamesTo
                     TableReference tableRefLowerCased = TableReference
                             .createUnsafe(table.getQualifiedName().toLowerCase());
                     if (!existingTablesLowerCased.contains(tableRefLowerCased)) {
-                        client.system_add_column_family(getCfForTable(table, metadata));
+                        filteredTables.put(table, metadata);
                     } else {
                         log.debug("Ignored call to create table ({}) that already existed (case insensitive).", table);
```

Comment:
Maybe modify this to say "Filtering out existing table {{}) yadda yadda"? Could potentially have another consumer that isn't createTables in the future?